### PR TITLE
Create loader

### PR DESCRIPTION
**Pull request by NikitaBondarevv**

### CODE REVIEW

File: src/components/preloader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styles from './styles.css'
+
+export const Preloader = () => (
+  <div className={styles.preloader}>
+    <div className={styles.loadingioSpinner}>
```
Comment (iliatcymbal) [inline src/components/preloader/index.tsx:5]: `loadingioSpinner` why `io`?

File: src/components/preloader/styles.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@keyframes ldio-smk3sd608te {
```
Comment (iliatcymbal) [inline src/components/preloader/styles.css:1]: what the heck is this name about?

File: src/pages/contacts/index.tsx [hunk 3]
```diff
@@ -3,7 +3,10 @@ import styles from './styles.css'
 export const Contacts = () => (
   <div className={styles.contacts}>
     <span>
-      Contact: <mark>nikita.bondarev1807@gmail.com</mark>
+      Contact:
+      <a href="mailto: nikita.bondarev1807@gmail.com" className={styles.email}>
```
Comment (iliatcymbal) [inline src/pages/contacts/index.tsx:7]: fix `href` https://www.w3docs.com/snippets/html/how-to-create-mailto-links.html

File: src/components/preloader/styles.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  border-top-color: transparent;
+  border-radius: 50%;
+}
+
+.loader div {
+  animation: ldio-smk3sd608te 1s linear infinite;
+  top: 100px;
+  left: 100px
+}
+
```
Comment (iliatcymbal) [inline src/components/preloader/styles.css:25]: why there are two different rules?

File: src/components/preloader/styles.css [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  height: 100%;
+  position: relative;
+  transform: translateZ(0) scale(1);
+  backface-visibility: hidden;
+  transform-origin: 0 0;
+}
+
+.loader div {
+  box-sizing: content-box;
+}
```
Comment (iliatcymbal) [inline src/components/preloader/styles.css:45]: a third rule for the same selector - why?

**Review by iliatcymbal — CHANGES_REQUESTED** [review]
an html and css of preloader is awful. Don't make copypast, use your brain. You need  only one tag for it (4 divs is too many)
Should be redone

File: src/pages/contacts/styles.css [hunk 6]
```diff
@@ -1,19 +1,53 @@
...
+.contacts textarea {
+  width: 450px;
+  height: 200px;
+  resize: none;
+  border-radius: 10px;
+}
+
+.contacts input {
+  font: 24px/24px "MyriadProLight";
+  padding: 5px 35px 5px 30px;
```
Comment (iliatcymbal) [inline src/pages/contacts/styles.css:29]: why there is such a huge left padding?

Comment (NikitaBondarevv) [reply src/pages/contacts/styles.css:29]: This is extra padding. There are enough 5 pixels on all sides. fixed.

**Pull request merged**